nixos/modules/installer/cd-dvd/sd-image.nix

```diff
@@ -203,7 +211,7 @@ in
       '';
     }) {};
 
-    boot.postBootCommands = ''
+    boot.postBootCommands = if config.sdImage.expandOnBoot then ''
```

Comment:
Might be better to use `mkIf config.sdImage.expandOnBoot` here.

Reply:
Done.